lnrpc/rest-annotations.yaml

```diff
@@ -133,6 +133,11 @@ http:
     - selector: lnrpc.Lightning.BakeMacaroon
       post: "/v1/macaroon"
       body: "*"
+    - selector: lnrpc.Lightning.ListMacaroonIDs
+      get: "/v1/macaroon/ids"
+    - selector: lnrpc.Lightning.DeleteMacaroonID
+      post: "/v1/macaroon/{root_key_id}"
```

Comment:
Just saw this now: Should be `delete: "/v1/macaroon/{root_key_id}"` instead. Then the `body` line below also isn't necessary.

Reply:
fixed.